src/Tutorials/Minimal-Orleans-Application.md

```diff
@@ -224,13 +224,13 @@ Within the appropriate bin directory (Debug/Release) there will be a number of l
 ## Troubleshooting
 - System.SystemException: 'Failed to start Orleans silo 'YOUR-HOST-NAME' as a Secondary node':
     - Problem: The silo is trying to start but does not find a the specified file to get the configuration.
-    - Solution: Set the property "Copy to Output Directory" to "Copy if newer", to set it, right click on the configuration file, then click on properites.
+    - Solution: Set the property "Copy to Output Directory" to "Copy if newer", to set it, right click on the configuration file, then click on properties.
     ```
 
 ## Further Reading
 
  - [List of Orleans Packages](http://dotnet.github.io/orleans/NuGets)
  - [Orleans Configuration Guide](http://dotnet.github.io/orleans/Orleans-Configuration-Guide/)
  - [Orleans Best Practices](http://research.microsoft.com/pubs/244727/Orleans%20Best%20Practices.pdf)
- - [Running in a Stand Alone Silo](http://dotnet.github.io/orleans/Step-by-step-Tutorials/Running-in-a-Stand-alone-Silo)
+ - [Running in a Stand Alone Silo](https://dotnet.github.io/orleans/Tutorials/Running-in-a-Stand-alone-Silo.html)
```

Comment:
Thanks a lot for reviewing this :+1: It's always a pleasure contributing to open source! I'll make the changes as soon as possible